.env-example

```diff
@@ -0,0 +1,14 @@
+SMTP_HOST=
+SMTP_USERNAME=
+SMTP_PASSWORD=
+SMTP_SENDER=
+JWT_SECRET=
+RECAPTCHA_SECRET_KEY=
+PUBLIC_BACKEND_URL=
+PUBLIC_FRONTEND_URL=
+MYSQL_ROOT_PASSWORD=rootpassword
+MYSQL_DATABASE=mydb
+MYSQL_USER=user
+MYSQL_PASSWORD=password
+BACKEND_FQDN=
+FROTNEND_FQDN=
```

Comment:
```suggestion
# Backend
SMTP_HOST=mail.example.com
SMTP_USERNAME=licensegate@example.com
SMTP_PASSWORD=mySMTPPassw0rd
SMTP_SENDER=LicenseGate <licensegate@example.com>
JWT_SECRET=secret
RECAPTCHA_SECRET_KEY=XXYYZZAABBCCDD
PUBLIC_BACKEND_URL=https://licensegate-api.example.com
PUBLIC_FRONTEND_URL=https://licensegate.example.com
MYSQL_ROOT_PASSWORD=rootpassword
MYSQL_DATABASE=mydb
MYSQL_USER=user
MYSQL_PASSWORD=password
BACKEND_FQDN=licensegate-api.example.com
FROTNEND_FQDN=licensegate.example.com

# Frontend
PUBLIC_BACKEND_URL=https://licensegate-api.example.com
PUBLIC_RECAPTCHA_SITE_KEY=XXYYZZAABBCCDD
PUBLIC_GOOGLE_AUTH_CLIENT_ID=none
```

Reply:
Also, should be fine in the sense of "what is actually included in the build":

According to Leoko and Sveltekit's documentation, only the envs starting with `PUBLIC` are included, which I can confirm on my test instance:

![image](https://github.com/user-attachments/assets/11e26093-9c49-41cb-93b8-37da3f02c019)